### DISCUSSION TITLE
按照官方教程无法编译C++版本，愁死我了！

### DISCUSSION THREAD

**Discussion by a1101773014** [PaddlePaddle/PaddleOCR#12368] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：win11(23H2)22631.3007，opencv4.9，cmake3.22.3, git2.16.1.windows.1， VS2019community
- 版本号/Version：Paddle：  PaddleOCR： 2.7
- 运行指令/Command Code：CMake
- 完整报错/Complete Error Message：

执行CMake时报错，愁死我了。
Failed to update submodules in:
    'D:/projects/cpp/PaddleOCR/deploy/cpp_infer/build/third-party/extern_autolog-src'

官方教程地址：
https://github.com/PaddlePaddle/PaddleOCR/blob/dygraph/deploy/cpp_infer/docs/windows_vs2019_build.md

![无标题](https://github.com/PaddlePaddle/PaddleOCR/assets/97845432/eda3abfe-726f-4ee1-9ffb-caa486564f73)
[详细信息.txt](https://github.com/PaddlePaddle/PaddleOCR/files/14106943/default.txt)

**Comment by GreatV** [comment]
看着像网络不行

**Comment by countryhotel** [comment]
楼主，你在安装  VS2019community 时，下图是选取的哪个选项？

![2019选取](https://github.com/PaddlePaddle/PaddleOCR/assets/154119759/07eb8cc6-03d5-46dd-b46f-e7b3a4720111)

**Comment by Hommoner** [comment]
建議直接使用FastDeploy, 可支援ppocr產出的模型
